priam/src/main/java/com/netflix/priam/backup/BackupRestoreUtil.java

```diff
@@ -144,4 +135,35 @@ public final boolean isFiltered(String keyspace, String columnFamilyDir) {
                             || includeFilter.get(keyspace).contains(columnFamilyName)));
         return false;
     }
+
+    /**
+     * Get all the backup directories for Cassandra.
+     *
+     * @param dataDirectory the location of the data folder.
+     * @param monitoringFolder folder where cassandra backup's are configured.
+     * @return Set of the path(s) containing the backup folder for each columnfamily.
+     * @throws IOException
+     */
+    public static ImmutableSet<Path> getBackupDirectories(
+            String dataDirectory, String monitoringFolder) throws IOException {
+        ImmutableSet.Builder<Path> backupPaths = ImmutableSet.builder();
+        Path dataPath = Paths.get(dataDirectory);
+        if (Files.exists(dataPath) && Files.isDirectory(dataPath))
```

Comment:
The `File.exists` call is redundant here: https://docs.oracle.com/javase/6/docs/api/java/io/File.html#isDirectory()